packages/config/src/json/json.config.ts

```diff
@@ -33,7 +33,8 @@ const Q = PLimit(10);
 
 export function guessIdFromUri(uri: string): string | null {
   const parts = uri.split('/');
-  const id = parts.pop();
+  let id = parts.pop();
+  if (uri.endsWith('/')) id = parts.pop();
```

Comment:
```suggestion
  const id = uri.endsWith('/') ? parts.at(-2) : parts.at(-1)
```

Avoids the double pop.